webapp/portlet/src/main/webapp/vue-apps/activity-stream/main.js

```diff
@@ -45,6 +45,9 @@ const urls = [
   `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.social.Webui-${lang}.json`,
 ];
 
+//used only in mobile statistics
+eXo.env.portal.mainApplicationName = 'Social';
```

Comment:
```suggestion
eXo.env.portal.mainApplicationName = 'activityStream';
```

It's better to add a name that you can add in i18N File and that is significative. To know if you want to use `social` (one single module name for all social apps) or `activityStream` + `peopleList` + `spaceList` + (moduleName per Application) ... could you please ask @Samuel-Renault-eXo or Morgan